.gitignore

```diff
@@ -15,3 +15,6 @@ target/
 ext-lib-src/
 .ensime
 .ensime_cache/
+moquette_store.mapdb
+moquette_store.mapdb.p
+moquette_store.mapdb.t
```

Comment:
just trying to apply "boyscout" pattern... these are files generated during tests.
I cannot be the only one who's bothered by seeing untracked files, and running `git clean -f` after every test :)